### ISSUE TITLE
[SCREENREADER]: JAWS + Chrome enables focus mode when we focus breadcrumb UL on page load

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#319]
## Issue
**UPDATE:** A related issue for another project led me to this finding: https://tink.uk/understanding-screen-reader-interaction-modes/

> To all intents and purposes, applications mode is the same as forms/focus mode – it causes the screen reader to pass keystrokes back through to the browser so they can fulfil their original purpose. For example, when the tablist and tab roles are used as part of the tab widget design pattern, using the tab key to move focus onto the first tab in the set causes a Windows screen reader to automatically switch into applications mode. From that point all the keyboard interaction is handled by the script. This does mean of course that the script driving the functionality of the widget has to be setup to handle keyboard interaction!

While testing some fixes for the HCA updates page as part of the pull request linked below, I noticed JAWS + Chrome (Windows) was switching to Forms mode on page load. Forms mode is generally expected to be enabled when users are interacting with form elements like inputs, text areas, radio buttons, and checkboxes. This was happening on a content page, and was an unexpected behavior.

## Related Items
* https://github.com/department-of-veterans-affairs/vets-website/pull/10003

## Acceptance Criteria
As a Chrome JAWS user, I want to be able to navigate through pages by pressing `DOWN_ARROW` and `UP_ARROW` without having to press `ENTER` to return to Browse mode on first load.

## Environment
* Windows 10
* Chrome latest
* JAWS 2019
* https://staging.va.gov/education/apply-for-education-benefits/application/1990/introduction
* https://staging.va.gov/health-care/apply/application/introduction
* Also likely to happen in other education benefits views

## Steps to Recreate
1. Start JAWS screen reader
2. Enter one of the previous IPs into Chrome on Windows
3. Listen for an audible "chirp" (this is JAWS switching to Forms mode)
4. When the breadcrumb is done being announced, press `DOWN_ARROW`
5. Verify you have not moved to the next DOM node, but your viewport has scrolled

## Possible Fixes (optional)
It appears this can be remediated by setting focus on the breadcrumb `<nav>` element instead of the `<ul>`. We will have to amend a number of `focusElement()` calls, and update some CSS to remove the yellow focus halo from the `<nav>` element.

```javascript
/* Switch the focusElement call to the <nav> className */
class IntroductionPageGated extends React.Component {
  componentDidMount() {
    focusElement('.va-nav-breadcrumbs');
  }
...
```

```css
nav[aria-label="Breadcrumb"]  {
  outline: 0;
}	
```

**Comment by alexpappasoddball** [comment]
@Karak888 I believe this one would be your team.  Can you confirm and move to your backlog if so?

**Comment by cvalarida** [comment]
:wave: Hey, @1Copenut. Since we (engineers) can't test in JAWS, can we get a clearer solution specification here? I don't even know where to start. I'd be happy to pair with you to figure out what we need to do, but we can't pick this ticket up and run with it as-written.

**Comment by k80bowman** [comment]
For this ticket, we only need to make sure this issue is not happening in the Breadcrumbs web component. Neither of the staging links above are using the breadcrumbs web component, so don't test using them. This page does use the Breadcrumbs web component, so you should be able to test this on it: https://staging.va.gov/education/education-letters/

**Comment by caw310** [comment]
@SarahKay8 , could you test and let us know if this is still a problem?  If it is still a problem, an engineer will need to fix.

**Comment by k80bowman** [comment]
@davidakennedy I hope you don't mind, I went ahead and assigned this to you so it's easier to find. As I noted in Slack, see [this comment](https://github.com/department-of-veterans-affairs/va.gov-team/issues/319#issuecomment-1218088790) for details about testing this. We just want to make sure this isn't an issue with our web component version of the breadcrumbs component.

**Comment by davidakennedy** [comment]
I tested this today, and the the web component breadcrumb doesn't have this issue any longer. This [intro page](https://staging.va.gov/education/apply-for-benefits-form-22-1990/introduction) with it was good to go. I loaded the page and the screen reader read it normally without going into forms mode. I was also able to traverse the breadcrumbs without issue.

The [first page of the form](https://staging.va.gov/education/apply-for-education-benefits/application/1990/applicant/information), which has the React component, still has the issue. I got the same behavior that Trevor mentions above.

**Comment by k80bowman** [comment]
Fantastic, thank you @davidakennedy! I am going to say that the solution for this problem is to migrate to the Breadcrumbs web component and close this ticket.
